src/SurveyService.Data/OptionRepository.cs

```diff
@@ -60,4 +66,19 @@ public Task DisactivateAsync(ICollection<DbOption> options)
 
     return _provider.SaveAsync();
   }
+
+  public async Task<bool> EditAsync(JsonPatchDocument<DbOption> patch, DbOption dbOption, Guid? modifiedBy = null)
+  {
+    if (patch is null || dbOption is null)
+    {
+      return false;
+    }
+    
+    patch.ApplyTo(dbOption);
+    dbOption.ModifiedBy = modifiedBy;
```

Comment:
dbOption.ModifiedBy = modifiedBy is null ? _httpContextAccessor.HttpContext.GetUserId() : modifiedBy;

if method would be used somewhere else

Reply:
Done